### DISCUSSION TITLE
20230218: Player volume resets to 100% on every play command if ReplayGain is active

### DISCUSSION THREAD

**Discussion by Sinescape** [rern/rAudio#1244] [Fixed Issues]
After updating to version 20230218 from an old 2021 version, the player volume gets automatically set to 100% everytime a track starts playing or resumes after pausing. This only happens if ReplayGain is active – the ReplayGain mode (auto/album/track) does not matter.

**Comment by cmh714** [comment]
probably has nothing to do with the issue, but I saw this when I updated...update went thru anyway, and I dont use that feature

curl -sfLO https://github.com/rern/rAudio-1/raw/main/install.sh
chmod 755 install.sh
./install.sh "r1
Update
20230218"

ln: failed to create symbolic link '/srv/http/data/camilladsp/active_config.yml': File exists

**Reply by rern** [reply]
Please try:
```sh
sed -i '/replaygain.conf/,/replay_gain_handler/ d' $dirsettings/player-conf.sh
player-conf.sh
```

**Reply by Sinescape** [reply]
That did the trick!
